frontend/src/firebase.js

```diff
@@ -0,0 +1,23 @@
+import firebase from "firebase/app";
+import "firebase/auth";
+import "firebase/firestore";
+
+// Your web app's Firebase configuration
+var firebaseConfig = {
+  apiKey: "AIzaSyB-VuNFq8GnXnGldqZMrIf6sfp9VFFtVNA",
+  authDomain: "astrocoders.firebaseapp.com",
+  databaseURL: "https://astrocoders.firebaseio.com",
+  projectId: "astrocoders",
+  storageBucket: "astrocoders.appspot.com",
+  messagingSenderId: "2750250289",
+  appId: "1:2750250289:web:a295f850f40944bc768410",
+  measurementId: "G-8ES6VWV752",
+};
```

Comment:
It's good practice to keep api keys (and other secrets) to a `.env` file that you don't commit to Git (add it to .gitignore for this), so people can't access it.